planet4-gutenberg-blocks.php

```diff
@@ -118,6 +118,7 @@
 	'planet4-blocks/submenu',
 	'planet4-blocks/take-action-boxout',
 	'planet4-blocks/timeline',
+	'planet4-blocks/spreadsheettable',
```

Comment:
Words in the block name should be separated by a dash, so that would make `spreadsheet-table` here.

Reply:
Please don't enabled it for pages post_type, only for campaigns.